### PR TITLE
updated controllers

### PR DESCRIPTION
**Pull request by JonasSchiessl**
Changed the types to capital, removed unnecessary login, register, forgot password controller functions.

### CODE REVIEW

**Review by meganfung38 — APPROVED** [review]
looks good

File: types/facility.ts
Comment (meganfung38) [inline types/facility.ts:1]: [{"Category": "Womens", "Coordinates": [], "Date": "2024-03-02T00:00:00.000Z", "Favorites": 1, "Name": "Restroom1", "Operations": "8am-9pm", "PictureURL": ["https://example.com/index.html"], "RatingAVG": 4.5, "Reports": 0, "Reviews": ["65e3cc149aa3eada64c4c739"], "Tags": ["Clean", "Feminine Products"], "_id": "65e3c7599aa3eada64c4c737", "coordinates": [64, 72]}]

this is what the controller function returns-- why is there a "Coordinates" and "coordinates", one should be deleted.

Comment (meganfung38) [reply types/facility.ts:1]: Also, how would I access the facility id if it isn't defined here in the type. I need the facility id to call the reviewController function.

**Review by meganfung38 — CHANGES_REQUESTED** [review]

**Review by meganfung38 — APPROVED** [review]
Looks good

**Comment by JonasSchiessl** [comment]
Can everyone review this one please.

**Review by k4yluh — DISMISSED** [review]
Looks good!

**Review by lindlands — DISMISSED** [review]
Looks good!

**Comment by dylantompkins** [comment]
```
The following packages should be updated for best compatibility with the installed expo version:
  @react-native-community/datetimepicker@8.0.0 - expected version: 8.0.1
  expo@51.0.5 - expected version: ~51.0.8
  expo-image-picker@15.0.4 - expected version: ~15.0.5
  jest-expo@51.0.1 - expected version: ~51.0.2
Your project may not work correctly until you install the expected versions of the packages.
```

**Review by dylantompkins — APPROVED** [review]
thank you for updating packages. LGTM

**Pull request merged**